src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -48,21 +48,25 @@ $spacing: 20px;
     height: fit-content;
   }
 
+  &__content {
+    flex: auto;
```

Comment:
This actually works? 😮

Reply:
Yes, it looks better for spaces with a short title/subtitle